fe/fe-core/src/main/java/org/apache/doris/common/util/PropertyAnalyzer.java

```diff
@@ -451,8 +453,27 @@ public static long analyzeTimeout(Map<String, String> properties, long defaultTi
         return timeout;
     }
 
+    public static Boolean analyzeUseLightSchemaChange(Map<String, String> properties) throws AnalysisException {
+        String useSchemaChange = "";
```

Comment:
Use boolean instead of String directly？

Reply:
I adapt this function using the similar logic as analyzeUniqueKeyMergeOnWrite below.